src/components/ShowList.css

```diff
@@ -0,0 +1,11 @@
+.check-item {
+  display: flex;
+}
+
+.check-item li {
+  list-style: none;
+}
+
+.completed {
+  color: gray;
+}
```

Comment:
I like the idea of adding this gray styling to the text when an item is ticked! I saw it when you previewed what you were doing on office hours and it looked great, but it doesn't seem to be working for me when I pull it down and tick an item.  🤔 I'm guessing it would be a class added as part of handling the checkbox click? Great idea 🎉

Reply:
Thank you @abi, I think we will add the property back